### PR TITLE
Create package publishable to pip

### LINKED ISSUE #18: Register this package with pip

### PR DESCRIPTION
**Pull request by Williangalvani**
This refactors the packages to follow Python coding guidelines before plublishing.

`brping/ping1d.py` and `brping/pingmessage.py` will be rewritten by the script in ping-protocol, updated in https://github.com/bluerobotics/ping-protocol/pull/106

Fix #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,15 @@ This library exposes all functionality of the device, such as getting profiles,
 
 ## Installing
 
-* `git clone --recursive http://github.com/bluerobotics/ping-python.git`
-* `cd ping-python`
-
+ - From source:
+```
```
Comment (patrickelectric) [inline README.md]:
```sh

**Comment by Williangalvani** [comment]
@jaxxzer can you look at this? I'd like to merge this one before https://github.com/bluerobotics/ping-protocol/pull/106

File: README.md [hunk 2]
```diff
@@ -26,9 +26,17 @@ This library exposes all functionality of the device, such as getting profiles,
...
+ - From source:
+```sh
+$ git clone --recursive http://github.com/bluerobotics/ping-python.git
+$ cd ping-python
+$ python setup.py install
+```
+ - Using 'pip'
+
+```sh
+$ pip install bluerobotics-python
```
Comment (jaxxzer) [inline README.md]: pip install bluerobotics-ping?

**Comment by jaxxzer** [comment]
I think it makes more sense to rebase this on the generated commits of  bluerobotics/ping-protocol#106.
I don't think it's a good idea to commit manual edits to the generated files here. I understand the workflow that you used to get to this point, but let me know if there is some problem to rebase this.

File: README.md (hunk 2 above)
Comment (Williangalvani) [reply README.md]: nice catch

**Comment by Williangalvani** [comment]
I'll do that then, no problem.

**Comment by Williangalvani** [comment]
@jaxxzer that is going to fail, because the CI script is going to try to move the files to brping/, which doesn't exist yet.
I can just drop the commits changing these files instead (but leave the ones changing the directory structure.)

**Comment by jaxxzer** [comment]
Ok @Williangalvani please take care of the force push here when you are comfortable with it, and I will proceed with the review. You can take care of the current fixups now or after I'm done with the review.

**Pull request merged**